openslides_backend/models/fields.py

```diff
@@ -120,46 +131,43 @@ def __init__(
 
 
 class RequiredOneToOneField(RelationMixin, IdField):
-
     on_delete = "protect"  # TODO: Enable cascade
     type = "1:1"
 
 
 class OneToOneField(RelationMixin, IdField):
-
     on_delete = "set_null"  # TODO: Enable cascade
     type = "1:1"
 
     def get_schema(self) -> Schema:
-        schema = super().get_schema()
-        schema["type"] = ["integer", "null"]
-        return schema
+        return self.extend_schema(super().get_schema(), type=["integer", "null"])
 
 
 class RequiredForeignKeyField(RelationMixin, IdField):
-
     on_delete = "protect"  # TODO: Enable cascade
     type = "1:m"
 
 
 class ForeignKeyField(RequiredForeignKeyField):
-
     on_delete = "set_null"  # TODO: Enable cascade
 
     def get_schema(self) -> Schema:
-        schema = super().get_schema()
-        schema["type"] = ["integer", "null"]
-        return schema
-
+        return self.extend_schema(super().get_schema(), type=["integer", "null"])
 
-class ManyToManyArrayField(RelationMixin):
-
-    type = "m:n"
 
+class ManyIdField(RelationMixin, IdField):
     def get_schema(self) -> Schema:
-        return dict(
-            description=self.description,
+        return self.extend_schema(
+            super().get_schema(),
             type="array",
             items={"type": "integer", "minimum": 1},
             uniqueItems=True,
         )
+
+
+class ManyToManyArrayField(ManyIdField):
+    type = "m:n"
+
+
+class ManyToOneArrayField(ManyIdField):
```

Comment:
Please remove or at least comment this type of field. I strongly feel that can not use it but build such relations always from the reverse side as ForeignKey.

Reply:
I needed this to be able to update the relations like `assignment_poll_default_group_ids` from the meeting side. As @FinnStutzenstein already said, it's better and wanted to have these as relations. I tried to define a `ForeignKey` on groups but with this I was not able to update it from the meeting side since the field does not exist there. I agree with you that this would be the cleaner solution, but I could not figure out what I needed to change to make it work. Please feel free to make the reverse relation fully functional, then we can remove this.

Reply:
Wait, reverse relations do currently not work in the action service??

Reply:
A ManyToOne-Field is not fully implemented in the relations.py logic. But I think is is possible to use the reverse relation by just adding a list of values to e. g. `assignment_poll_default_group_ids`. This case should be handles e. g. [here](https://github.com/OpenSlides/openslides-backend/blob/master/openslides_backend/action/relations.py#L254).

Reply:
Yes, that is what I wanted to do initially. If I add a group model with

    used_as_motion_poll_default_id = fields.ForeignKeyField(
        to=Collection("meeting"), related_name="motion_poll_default_group_ids"
    )

and give `motion_poll_default_group_ids` in the create/update schema of a meeting, the backend fails to boot with `ValueError: motion_poll_default_group_ids is not a field of meeting`. That's why I wrote the `ManyToOneField`. @normanjaeckel If you fix the relations, we can remove it again.

Reply:
Please merge #87 first and then set this pull request on top. Then the problem should be solved.

Reply:
Does not work right now. I added a test `test_settings` which fails because of the bugged relation handling. The code seems to think that the `1` side of the relation is an array and the `m` side is a single id...
Other way to reproduce another error: startup the main repos setup and try to update the default group ids in the settings, it fails on https://github.com/OpenSlides/openslides-backend/blob/19de8c82603df7e35fdd43d7823cf38dbc9113d3/openslides_backend/action/relations.py#L283 because the value is not a list.